etc/microsoft/create-blank-vhd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.7.1-alpine
+FROM golang:alpine
```

Comment:
Looks like we upgraded to a newer go image. Let's make sure we keep referencing the image by a version tag, even if its the latest tag.